core/grid.ts

```diff
@@ -97,21 +97,21 @@ export class Grid {
   update(scale: number) {
     this.scale_ = scale;
     // MSIE freaks if it sees a 0x0 pattern, so set empty patterns to 100x100.
```

Comment:
this says it's for IE but it affects everything rn. seems like a reasonable default for everyone, so maybe update the comment? but you can also ignore; this isn't related

Reply:
Ignoring as unrelated but I added it to the list of places where we do things because of IE: https://github.com/google/blockly/issues/6325#issuecomment-1224648805